core/src/main/java/tech/tablesaw/joining/DataFrameJoiner.java

```diff
@@ -442,7 +431,9 @@ public Table leftOuter(Table... tables) {
   public Table leftOuter(boolean allowDuplicateColumnNames, Table... tables) {
     Table joined = table;
     for (Table table2 : tables) {
-      joined = leftOuter(table2, allowDuplicateColumnNames, joinColumnNames);
+      joined =
+          joinInternal(
```

Comment:
Why do you have to use `joinInternal` here? Does that mean `leftOuter(table2, allowDuplicateColumnNames, joinColumnNames)` is still broken?

Reply:
No. `leftOuter(table2, allowDuplicateColumnNames, joinColumnNames)` works well when joining field `table` with another table `table2`. 

https://github.com/jtablesaw/tablesaw/blob/627be4b660e6ff12eec3c06b7a2c9e353698f911/core/src/main/java/tech/tablesaw/joining/DataFrameJoiner.java#L488-L490

Problem is that field `table` is final and is assigned in the constructor. If we want to join multi-talbes, we must use the join result `joined` in the next iteration. Hence I choose to use `joinInternal`.